rest-api/src/main/java/org/jboss/pnc/client/Configuration.java

```diff
@@ -55,4 +77,67 @@ public BasicAuth(String username, String password) {
         }
     }
 
+    public static final class ConfigurationBuilder {
```

Comment:
Do you know that you can combine custom with lombok generated builder?

Reply:
Forgot about that, thanks updating it.